main.tex

```diff
@@ -1154,22 +1160,22 @@ \section*{Conclusion}
 Here, we have developed a specialization of PROV-DM focusing on entities and activities.
 It builds on an earlier PROV-DM specialization in which \SM{}, \SE{}, \SD{}, and \WD{} have been identified as crucial entities of simulation studies~\cite{Ruscheinski2018}.
 Additionally, we have taken knowledge of modeling and simulation life cycles~\cite{Balci2012} into account and identified the \RQ{}, \As{}, \Rs{}, the \QM{} to be important ingredients of the provenance of simulation models.
-We are also distinguishing between \BSM{}, \CSM{}, \VSM{}, and \ASM{} activities and connect the entities and activities by using the relations \textit{wasGeneratedBy} and \textit{used}.
-In our definitions of the entities and activities, we have aimed at the minimal level of detail, or granularity, of the provenance graph to understand the course of a simulation study.
-We have also kept the necessary metadata of the entities and activities minimal to convey both the main idea of the simulation study and the content of each entity and activity.
+We also distinguish between \BSM{}, \CSM{}, \VSM{}, and \ASM{} activities and connect the entities and activities by using the relations \textit{wasGeneratedBy} and \textit{used}.
+In our definitions of the entities and activities, we aimed at achieving the minimal level of detail, or granularity, of the provenance graph to understand the course of a simulation study.
+We also kept the necessary metadata of the entities and activities to a minimum to convey both the main idea of the simulation study and the content of each entity and activity.
 For storing, visualizing, and querying the provenance information, we have created the web-based tool \webprov{} that allows for each entity and activity to store (customized) metadata and references.
 
 % Insights into the Wnt family
 In order to examine our specialization of PROV-DM, the extensive analysis of 19 simulation studies of the canonical \wnt{} signaling pathway provided a suitable case study.
-We were able to explicitly show that most studies are connected to one or more other \wnt{} simulation studies, using (parts of) their simulation models, but also various data from wet-lab studies.
-Our result marks the outstanding role of the \wnt{} simulation model by \lee{}~\cite{Lee2003} as the origin for most other models in our survey.
+We were able to explicitly show that most studies are connected to one or more other \wnt{} simulation studies, using (parts of) their simulation models, in addition to various data from wet-lab studies.
+Our results show the outstanding role of the \wnt{} simulation model by \lee{}~\cite{Lee2003} as the origin for most other models in our survey.
```

Comment:
Not 100% sure about this change but I think "results show" is a bit clearer than "result marks"